### DISCUSSION TITLE
Sonoff POW R2 to monitor power only

### DISCUSSION THREAD

**Discussion by novellaip** [RavenSystem/esp-homekit-devices#1555] [Show and tell]
Jumping wire from CF pin to I05 pin do the trick.
![B1CA7323-8F07-4072-B6B7-8312E5164EA1](https://user-images.githubusercontent.com/69299111/132541032-fb93440e-165e-4871-b800-cb05c30b76af.jpeg)
![ADCF6341-A419-44B5-AFB2-018B49CDCC6C](https://user-images.githubusercontent.com/69299111/132541099-d8a518f7-78e6-4847-96a0-94e10e1569ac.jpeg)
![34CAFE23-54DE-489F-B2BF-AD96000B7A1D](https://user-images.githubusercontent.com/69299111/132541107-5373d1c0-3d93-4dde-ab6a-0ad7a926dc09.png)
![9B503320-EDC4-4C2B-A4A8-168F42D6ED5B](https://user-images.githubusercontent.com/69299111/132541110-4f527171-02e5-436b-a422-00913d2db9fd.jpeg)
![1FF6067C-17C4-41D7-AA88-322A2873F1FA](https://user-images.githubusercontent.com/69299111/132541116-154b28a2-8c2b-4ee7-976f-cdfbdac572b1.png)

**Comment by nickforev** [comment]
Hi. On/Off work? JSON please...

**Comment by novellaip** [comment]
{
	"c": {
		"l": 13,
		"b": [{
			"g": 0,
			"t": 5
		}]
	},
	"a": [{
		"t": 2,
		"s": 0,
		"0": {
			"r": [{
				"g": 12,
				"v": 0
			}]
		},
		"1": {
			"r": [{
				"g": 12,
				"v": 1
			}]
		},
		"b": [{
			"g": 0,
			"t": 1
		}],
		"es": [{
			"t": 75,
			"j": 2,
			"dt": [5],
			"pf": 4.4445
		}]
	}]
}

**Comment by i3laze** [comment]
HAA 10.0 is out. Have you tried new I2C feature?

got two POW R2 and zero free time(
